### PR TITLE
Development/foysal

### PR DESCRIPTION
**Pull request by Abdullahfoysal**
covid19 story youtube video streaming added fully

### CODE REVIEW

File: Covid19Shahajjo/app/src/main/java/com/example/covid19shahajjo/activities/Covid19StoryActivity.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+       /* Data Schema :
+       root:
+        video:
+            autoGenerated Id:
+                            title: about the video or selialization
+                            url:  zx6uMCoW2gQ  //https://www.youtube.com/watch?v=zx6uMCoW2gQ
+
+
+        youTubeVideos.addYoutubeVideoData(databaseReference,"about the video or selialization","zx6uMCoW2gQ");
+
```
Comment (rafiulgits) [inline Covid19Shahajjo/app/src/main/java/com/example/covid19shahajjo/activities/Covid19StoryActivity.java]: remove comments

File: Covid19Shahajjo/app/src/main/java/com/example/covid19shahajjo/activities/Covid19StoryActivity.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        //update url with presentTitle
+//        youTubeVideos.updateYoutubeVideoUrlData();
+
+        //update title+ url with presentTitle
+        youTubeVideos.updateYoutubeVideoData(databaseReference,"Empty","About Corona virus","PSnSo9kYlH4");*/
+
+
+       /*********************************GET VIDEO INFO FROM FIREBASE*********************************************/
+
```
Comment (rafiulgits) [inline Covid19Shahajjo/app/src/main/java/com/example/covid19shahajjo/activities/Covid19StoryActivity.java]: remove all comments from here.

File: Covid19Shahajjo/app/src/main/java/com/example/covid19shahajjo/activities/youtubeFullScreenActivity.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.example.covid19shahajjo.R;
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.PlayerConstants;
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.YouTubePlayer;
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.listeners.AbstractYouTubePlayerListener;
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.views.YouTubePlayerView;
+
+public class youtubeFullScreenActivity extends AppCompatActivity {
```
Comment (rafiulgits) [inline Covid19Shahajjo/app/src/main/java/com/example/covid19shahajjo/activities/youtubeFullScreenActivity.java]: class name should start with Capital letter

File: Covid19Shahajjo/app/src/main/java/com/example/covid19shahajjo/activities/Covid19StoryActivity.java (hunk 1 above)
Comment (Abdullahfoysal) [reply Covid19Shahajjo/app/src/main/java/com/example/covid19shahajjo/activities/Covid19StoryActivity.java]: done vhai

File: Covid19Shahajjo/app/src/main/java/com/example/covid19shahajjo/activities/youtubeFullScreenActivity.java (hunk 3 above)
Comment (Abdullahfoysal) [reply Covid19Shahajjo/app/src/main/java/com/example/covid19shahajjo/activities/youtubeFullScreenActivity.java]: okk bhai .
updating

**Pull request closed without merging**